Tempura/Core/ViewControllerWithLocalState.swift

```diff
@@ -175,7 +179,7 @@ open class ViewControllerWithLocalState<V: ViewControllerModellableView & UIView
  }
   
   /// This method is called every time the local state changes.
-  private func localStateDidChange() {
+  private func handleLocalStateChange() {
```

Comment:
Renamed to avoid confusion with the newly added `localStateDidChange` open method.